dev/CommandBarFlyout/InteractionTests/CommandBarFlyoutTests.cs

```diff
@@ -188,6 +188,39 @@ public void CanTapOnSecondaryItems()
             }
         }
 
+        [TestMethod]
+        public void CanTapOnSecondaryItemWithFlyoutWithoutClosing()
+        {
+            using (var setup = new CommandBarFlyoutTestSetupHelper())
+            {
+                Button showCommandBarFlyoutButton = FindElement.ByName<Button>("Show CommandBarFlyout with sub-menu");
+                ToggleButton isFlyoutOpenCheckBox = FindElement.ById<ToggleButton>("IsFlyoutOpenCheckBox");
+
+                Action openCommandBarAction = () =>
```

Comment:
Is there a specific reason we are defining an action for this?

Reply:
This was probably copied from other test methods where we run the action multiple times.

Reply:
Fair enough.

Reply:
The other tests in the file do it in order to reuse the action. For this tests its only used once and isn't really needed. But I think leaving it as is is probably fine.